docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Get and set recurrence in an Outlook add-in
+description: Get and set various recurrence properties of an item in an Outlook add-in.
+author: elizs
+ms.topic: article
+ms.technology: office-add-ins
+ms.date: 12/05/2018
+ms.author: elizs
+---
+
+# Get and set recurrence
+
+Sometimes you need to create and update a recurring appointment, such as a weekly status meeting for a team project or a yearly birthday reminder. You can use the JavaScript API for Office to manage the recurrence patterns of an appointment series in your add-in.
+
+## Available recurrence patterns
+
+To configure the recurrence pattern, you need to combine the [recurrence type](/javascript/api/outlook_1_7/office.mailboxenums.recurrencetype) and its applicable [recurrence properties](/javascript/api/outlook_1_7/office.recurrenceproperties) (if any).
```

Comment:
For accessibility, instead of table headings, consider adding a lead-in sentence before each table. "The following table contains recurrence types and their applicable properties."